modules/ossm-add-project-member-roll-resource-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-add-project-member-roll-resource-cli_{context}"]
+= Adding or removing projects from the service mesh using ServiceMeshMemberRoll resource with the CLI
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
+* The `ServiceMeshMemberRoll` resource is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+.Prerequisites
+
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* Name of the project with the `ServiceMeshMemberRoll` resource.
+* Names of the projects you want to add or remove from the mesh.
+* Access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Log in to the {product-title} CLI.
+
+. Edit the `ServiceMeshMemberRoll` resource.
++
+[source,terminal]
+----
+$ oc edit smmr -n <controlplane-namespace>
+----
++
+
+. Modify the YAML to add or remove projects as members.  You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+
++
+.Example servicemeshmemberroll-default.yaml
+
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system #control plane project
+spec:
+  members:
+    # a list of projects joined into the service mesh
+    - your-project-name
+    - another-project-name
+----
```

Comment:
Step 4 is missing. It should instruct the user to save the file and exit the editor. The SMMR is updated only after the user does this.